components/remote_settings/src/lib.rs

```diff
@@ -24,6 +28,48 @@ use storage::Storage;
 
 uniffi::setup_scaffolding!("remote_settings");
 
+/// The `AppContext` object represents the parameters and characteristics of the
+/// consuming application. For `remote-settings`, it is used to filter locally stored `records`.
```

Comment:
Is `remote-settings` referring to something specific here? Otherwise I would suggest we spell Remote Settings everywhere we refer to the service/feature

Reply:
I wasn't sure either. When spelling `remote-settings`, I mean the component.